rex_lee.md

```diff
@@ -1,17 +1,15 @@
-## 07/04 - 07/08 ##
+## 07/11 - 07/15 ##
 
 ### This week ###
 [firefox]
  - Working bugs:
    - Bug 1270006 - Replace the downloads remaining time on the downloads button with a downloading icon 
    - Bug 1282689 - Show an alert mark on badges if there are exceptional status to download files. 
-[TV]
- - Code reviewing
 
 ### Last weeks ###
 [firefox]
  - Working bugs:
-   - Bug 1261133 - Style editor doesn't remove ".moz-styleeditor-transitioning" if I close it in the middle of transition, so there're infinite transitions (landed)
    - Bug 1270006 - Replace the downloads remaining time on the downloads button with a downloading icon 
+   - Bug 1282689 - Show an alert mark on badges if there are exceptional status to download files. 
```

Comment:
You didn't update the bug for 3 weeks. Please write something on the bug when you claim you were working on it. Thanks.